### PR TITLE
Add workflow to generate RPM package

### PR DESCRIPTION
**Pull request by adrianfusco**
- Add **packaging** job that will build the RPM package in the workflow and publish it as an artifact
- Add the base SPEC file 

Note: keep in mind this is the base. Once other PRs are merged and we advance I'll adapt the SPEC file (e.g. add a way to generate releases using something like this https://github.com/actions/create-release )

### CODE REVIEW

**Review by cescgina — APPROVED** [review]
LGTM

File: packing_manager_specs/RPM/rpmbuild/SPECS/gz-beat-shipper.spec
Comment (jsanemet) [inline packing_manager_specs/RPM/rpmbuild/SPECS/gz-beat-shipper.spec]: Maybe leave this up to the user. I do not know many daemons that get to start on their own.

File: packing_manager_specs/RPM/rpmbuild/SPECS/gz-beat-shipper.spec
Comment (jsanemet) [inline packing_manager_specs/RPM/rpmbuild/SPECS/gz-beat-shipper.spec]: This and the one defined above do not match.

**Review by jsanemet — APPROVED** [review]
Looks good. You can take a look at my comments before merging.

File: packing_manager_specs/RPM/rpmbuild/SPECS/gz-beat-shipper.spec
Comment (adrianfusco) [reply packing_manager_specs/RPM/rpmbuild/SPECS/gz-beat-shipper.spec]: Yes, leaving it up or not to the user are both valid. I've deleted the commented calls in the post sections.

File: packing_manager_specs/RPM/rpmbuild/SPECS/gz-beat-shipper.spec
Comment (adrianfusco) [reply packing_manager_specs/RPM/rpmbuild/SPECS/gz-beat-shipper.spec]: I didn't realize, thanks. I've used the variable.

**Pull request merged**
